filebeat/crawler/prospector_log.go

```diff
@@ -270,11 +272,11 @@ func (p *ProspectorLog) checkExistingFile(h *harvester.Harvester, newFile *input
 func (p *ProspectorLog) continueExistingFile(h *harvester.Harvester, previousFile string) {
 	logp.Debug("prospector", "Launching harvester on renamed file. File rename was detected: %s -> %s", previousFile, h.Path)
 
-	lastinfo := p.harvesterStats[previousFile]
-	h.Stat.Continue(&lastinfo)
+	lastinfo := p.harvesterStates[previousFile]
+	h.State.Continue(&lastinfo)
 
 	// Update state because of file rotation
-	h.UpdateState()
+	h.SendStateUpdate()
```

Comment:
Currently yes :-( Or lets say it is triggered by the prospector. It is still the harvester that sends the state information.